modules/installation-special-config-raid-intel-vroc.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-special-config-raid-intel-vroc_{context}"]
+== Configuring an Intel(R) Virtual RAID on CPU (VROC) data volume
+
+Intel(R) VROC is a type of hybrid RAID, where some of the maintenance is offloaded to the hardware, but appears as software RAID to the operating system.
+
+The following procedure configures an Intel(R) VROC-enabled RAID1.
+
+.Procedure
```

Comment:
One note here that the Intel(R) Volume Management Device (VMD) controller needs to be enabled in the system firmware.

Should we have a prerequisite here of something like:

> You have a system with Intel(R) Volume Management Device (VMD) enabled.

?

Reply:
yes, thank you!